dcrd.go

```diff
@@ -149,6 +150,25 @@ func dcrdMain() error {
 		return nil
 	}
 
+	// Load the UTXO database.
+	lifetimeNotifier.notifyStartupEvent(lifetimeEventDBOpen)
```

Comment:
Good point.  I decided to just remove the event currently but can add it back if it would be useful in any way for a consumer to know when the UTXO database is being opened or closed.

I did notice when reviewing the related code that at one point, the [lifetimeEventTicketDB event was removed](https://github.com/decred/dcrd/commit/a9234850e7ebe24144e75b317e4102722f80de94#diff-3f280ea2540b796e617e2469da80fa44399c873d647db3a3c53e3566d3d813a0) which resulted in `lifetimeEventP2PServer` changing from `2` to `1` (maybe unintentionally?) and the [comments in the current code](https://github.com/decred/dcrd/blob/0b39e530e7b7b7126b1677c85a6023ac30c07f5b/ipc.go#L117-L146) don't reflect this change.

Reply:
Good catch.  We'll want to get those comments updated to match reality.  I suspect it's fine to have no event here, but I pinged the dcrwallet and Decrediton leads since this is something solely used by those projects, so they might want an event for it.

Reply:
Sounds good, thanks.  I'll wait for confirmation on whether we want an event for this, and in either case, I can update the comments to be accurate.

Reply:
It's not used by dcrwallet at all (dcrwallet does not spawn a dcrd process, so it has no ipc pipe to read these events).  It might be used by decrediton but unsure.  Originally these events were added so paymetheus could show feedback about what state dcrd was currently in after it had spawned the process, and to know exactly when it was ready to start dcrwallet.

Reply:
Thanks for confirming.

Also, Decrediton doesn't use them either, so the latest iteration of this PR without the event is good to go.

Reply:
Great, thanks for confirming!